src/generic_actor.cr

```diff
@@ -4,6 +4,7 @@ module GenericActor
   VERSION = "0.1.0"
 
   macro included
```

Comment:
Looks like we don`t need this `macro included` 
Variables and functions will be added to includer class any way